packages/backend-for-frontend/src/routers/catalogRouter.ts

```diff
@@ -316,6 +317,34 @@ const catalogRouter = (
         }
       }
     )
+    .post(
+      "/eservices/:eServiceId/descriptors/:descriptorId/agreementApprovalPolicy/update",
+      async (req, res) => {
+        const ctx = fromBffAppContext(req.ctx, req.headers);
+        try {
+          await catalogService.updateAgreementApprovalPolicy(
+            unsafeBrandId(req.params.eServiceId),
+            unsafeBrandId(req.params.descriptorId),
+            req.body,
+            ctx
+          );
+          return res.status(204).send();
+        } catch (error) {
+          const errorRes = makeApiProblem(
+            error,
+            updateAgreementApprovalPolicyErrorMapper,
```

Comment:
```suggestion
            emptyErrorMapper,
```
Since there's no processing on the BFF we should just let the errors from the process through without any modification